exporters/otlp/common/src/main/java/io/opentelemetry/exporter/internal/grpc/DefaultGrpcExporterBuilder.java

```diff
@@ -167,7 +172,7 @@ public GrpcExporter<T> build() {
 
     Codec codec = compressionEnabled ? new Codec.Gzip() : Codec.Identity.NONE;
     MarshalerServiceStub<T, ?, ?> stub =
-        stubFactory.apply(channel).withCompression(codec.getMessageEncoding());
+        stubFactory.apply(channel, authorityOverride).withCompression(codec.getMessageEncoding());
```

Comment:
BTW, while many call options are available on the stub itself, e.g. `stub.withDeadline`, authority isn't probably because it's an experimental API.